ledger/participant-integration-api/src/main/scala/platform/store/backend/common/EventStorageBackendTemplate.scala

```diff
@@ -419,13 +419,16 @@ abstract class EventStorageBackendTemplate(
         (wildcardPartiesClause ::: filterPartiesClauses).mkComposite("(", " or ", ")")
 
       def selectFrom(table: String, selectColumns: String) = cSQL"""
-        SELECT
+        (SELECT #$selectColumns, event_witnesses, command_id FROM ( SELECT
```

Comment:
We need to wrap a inner most subquery that is doing `order by` in another subquery because on Oracle subqueries used with `union all` cannot contain `order by` clause

Reply:
Sure